models/spleen_deepedit_annotation/configs/inference.json

```diff
@@ -60,49 +61,79 @@
+        },
+        {
+            "_target_": "ScaleIntensityRanged",
+            "keys": "image",
+            "a_min": -175,
+            "a_max": 250,
+            "b_min": 0.0,
+            "b_max": 1.0,
+            "clip": true
+        }
+    ],
+    "auto_seg_transforms": [
+        {
+            "_target_": "Resized",
+            "keys": "image",
+            "spatial_size": "@spatial_size",
+            "mode": "area"
+        },
+        {
+            "_target_": "DiscardAddGuidanced",
+            "keys": "image",
+            "label_names": "@label_names",
+            "number_intensity_ch": "@number_intensity_ch"
+        }
+    ],
+    "deepedit_transforms": [
+        {
+            "_target_": "AddGuidanceFromPointsDeepEditd",
+            "ref_image": "image",
+            "guidance": "guidance",
+            "label_names": "@label_names"
+        },
+        {
+            "_target_": "Resized",
+            "keys": "image",
+            "spatial_size": "@spatial_size",
+            "mode": "area"
+        },
+        {
+            "_target_": "ResizeGuidanceMultipleLabelDeepEditd",
+            "guidance": "guidance",
+            "ref_image": "image"
+        },
+        {
+            "_target_": "AddGuidanceSignalDeepEditd",
+            "keys": "image",
+            "guidance": "guidance",
+            "number_intensity_ch": "@number_intensity_ch"
+        }
+    ],
+    "extra_transforms": [
+        {
+            "_target_": "EnsureTyped",
+            "keys": "image"
+        }
+    ],
     "preprocessing": {
         "_target_": "Compose",
-        "transforms": [
-            {
-                "_target_": "LoadImaged",
-                "keys": "image",
-                "reader": "ITKReader"
-            },
-            {
-                "_target_": "EnsureChannelFirstd",
-                "keys": "image"
-            },
-            {
-                "_target_": "Orientationd",
-                "keys": "image",
-                "axcodes": "RAS"
-            },
-            {
-                "_target_": "ScaleIntensityRanged",
-                "keys": "image",
-                "a_min": -175,
-                "a_max": 250,
-                "b_min": 0.0,
-                "b_max": 1.0,
-                "clip": true
-            },
-            {
-                "_target_": "Resized",
-                "keys": "image",
-                "spatial_size": "@spatial_size",
-                "mode": "area"
-            },
-            {
-                "_target_": "DiscardAddGuidanced",
-                "keys": "image",
-                "label_names": "@label_names",
-                "number_intensity_ch": "@number_intensity_ch"
-            },
-            {
-                "_target_": "EnsureTyped",
-                "keys": "image"
-            }
-        ]
+        "transforms": "$@preprocessing_transforms + (@deepedit_transforms if @use_click else @auto_seg_transforms) + @extra_transforms"
```

Comment:
I remember @diazandr3s said we also need to add flag for the postprocessing logic, but don't any change for it.
Is it correct?

Thanks.

Reply:
Hi @Nic-Ma,

Postprocessing depends on what we define as preprocessing. @yiheng-wang-nv proposed a simpler solution to this which works as expected.

Thanks!